extension/src/utils/workspace.ts

```diff
@@ -93,7 +93,12 @@ export async function checkForExistingAppHostPathInWorkspace(terminalProvider: A
     
     let proc: ChildProcessWithoutNullStreams;
     new Promise<AppHostProjectSearchResult>((resolve, reject) => {
-        proc = spawnCliProcess(terminalProvider, 'aspire', ['extension', 'get-apphosts'], {
+        const args = ['extension', 'get-apphosts'];
+        if (process.env.ASPIRE_CLI_STOP_ON_ENTRY === 'true') {
```

Comment:
Add a comment here that explains how you use this to effectively debug the extension/CLI integration. You would need:

1. Open a shell and set the environment variable.
2. Open the extension code in VSCode.
3. Launch the VSCode instance in the debugger.
4. Set the path to the artifacts/ version of the CLI
5. Open the aspire repo in another instance of VSCode
6. Trigger an extension operation
7. Attach the debugger inside the instance of VSCode.